src/Select.js

```diff
@@ -147,6 +162,22 @@ class Select extends React.Component {
 		this.toggleTouchOutsideEvent(false);
 	}
 
+	handleAutoFillAnimation ({ target, animationName }) {
+		if (animationName === 'onAutoFillStart') {
+			this.setAutofill();
+		} else {
+			this.clearAutofill();
+			this.clearAutofilled();
+		}
+	}
+
+	addAutoFillListener () {
+		// hack to receive -webkit-autofill event
+		// @see https://medium.com/@brunn/detecting-autofilled-fields-in-javascript-aed598d25da7
+		this.inputField && this.inputField.addEventListener('animationstart', this.handleAutoFillAnimation, false);
```

Comment:
I prefer using `if (this.inputField) ...` for this kind of expression. If that's common in this repo, I guess its fine to leave as-is.

Reply:
nevermind